packages/terra-clinical-item-display/themes/orion-fusion-theme/orion-fusion-theme.scss

```diff
@@ -0,0 +1,14 @@
+:global {
```

Comment:
why are these theme files globally scoped?

Reply:
Theme aggregation scopes theme files to root or theme-name. Scoping these globally avoids a second scope layer. This should not be an issue - with the exception of the generated files, these component theme files are not imported by the component. Someone could manually import these, but that's on them.

Reply:
Global is our default, though it doesn't hurt to be explicit.